map/framework.cpp

```diff
@@ -2472,7 +2474,7 @@ void Framework::SetRouterImpl(RouterType type)
 
     router.reset(
         new CarRouter(m_model.GetIndex(), countryFileGetter,
-                      SingleMwmRouter::CreateCarRouter(m_model.GetIndex())));
+                      SingleMwmRouter::CreateCarRouter(m_model.GetIndex(), m_routingSession)));
```

Comment:
Yes. That's right. When we construct one of routers (car router) we pass traffic info getter (m_routingSession implements it).